### DISCUSSION TITLE
Can I connect multiple DICOM devices to one MIG ?

### DISCUSSION THREAD

**Discussion by jordimassaguerpla** [Project-MONAI/monai-deploy-informatics-gateway#376] [General]
Can I have multiple orthanc servers all connected to one single MIG? 

thanks

**Comment by mocsharp** [comment]
Hi @jordimassaguerpla, yes, you may register multiple Orthanc servers or any DICOM servers with MIG as long as you register each as a DICOM source.

Please refer to the [documentation ](https://monai.io/monai-deploy-informatics-gateway/setup/setup.html#enable-incoming-associations)for more information.
